### ISSUE TITLE
Styling: Smaller Marker Icons

### ISSUE THREAD

**Issue by MappingKat** [nationalparkservice/npmap-builder#194]
Don't mean to get nit-picky, but is there a way we can offer smaller marker sizes?  It would be nice to have some markers fade in the background so that other markers could stick out more.  So, by either having circular markers (like in cartocss marker-file) or just smaller-sized markers. 

![screen shot 2015-02-27 at 1 23 34 pm](https://cloud.githubusercontent.com/assets/1077436/6420522/fcca4ebc-be83-11e4-85df-3605a7d0a8df.png)

**Comment by makella** [comment]
One idea is to have the ability to choose either a teardrop marker or a simple point marker and add that to a 'marker type' drop-down.

**Comment by nateirwin** [comment]
What about this?

![map___builder___insidemaps___national_park_service](https://cloud.githubusercontent.com/assets/10443/6420753/3543a70a-be86-11e4-9c1b-52fea59fb471.jpg)

**Comment by makella** [comment]
I'm referring more to having more options to symbolize points.  So in a addition to Small, Medium, Large teardrop markers you could have the same option for a simple circular marker.

**Comment by MappingKat** [comment]
I am using the small options currently in that screenshot, but I think we need something smaller for zooms 5 and lower. 

I like that option.  Another option is to add one more size, as you mentioned, or change the size of the markers according to zoom...

There are def workarounds, but I think its worth discussing.

**Comment by nateirwin** [comment]
Ah, I see. Right now we are relying on the options that Mapbox makes available through its API: https://www.mapbox.com/developers/api/static/#markers-custom. But we can setup our own Maki server and then add some other "base" shapes and sizes to it.

**Comment by makella** [comment]
Yeah, I think that would be a good option.. we can add a few more shapes/sizes.
